.github/workflows/update-superchain.yml

```diff
@@ -0,0 +1,35 @@
+name: Update Superchain Config
+
+on:
+  schedule:
+    - cron: '0 3 * * 1'
+    - cron: '0 3 * * 4'
```

Comment:
let's do weekly and dispatch
    # Run weekly
    - cron: "0 0 * * SUN"
  workflow_dispatch: